Gopkg.lock

```diff
@@ -186,10 +174,10 @@
   version = "0.0.3"
 
 [[projects]]
+  branch = "roman/packages"
```

Comment:
can you make a release?

Reply:
I sure will once I've merged that reporting PR.